src/main/java/com/znsio/rpi/utils/ScreenShotManager.java

```diff
@@ -0,0 +1,81 @@
+package com.znsio.rpi.utils;
+
+import org.apache.commons.lang3.exception.ExceptionUtils;
+import org.apache.log4j.Logger;
+import org.openqa.selenium.OutputType;
+import org.openqa.selenium.TakesScreenshot;
+import org.openqa.selenium.WebDriver;
+
+import javax.imageio.ImageIO;
+import java.awt.*;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
+
+public class ScreenShotManager {
+    private static final String directoryPath = System.getProperty("user.dir") + File.separator +
+            "TestReport/Screenshots";
+    private static final Logger LOGGER = Logger.getLogger(ScreenShotManager.class.getSimpleName());
+
+    public static File captureScreenShot(WebDriver webDriver) {
```

Comment:
will this be public, or package-protected?

Reply:
Kept this public so that it can be used by the consumers for just capturing screenshots.